src/Forms/__tests__/Dropdown.test.tsx

```diff
@@ -414,8 +420,7 @@ describe('Dropdown', function () {
       deepStrictEqual(optionsFound, optionsWithDefaults);
     });
     it('the label value is hidden in the UI', function () {
```

Comment:
One last minor note: Can we update the name of this test to be something like "The label is still accessible in the DOM", since we're no longer testing whether it's hidden? (also applies in `TextInput.test.tsx`)

Reply:
good point, thanks.